ui/components/AccountItem/AccountItemOptionsMenu.tsx

```diff
@@ -43,10 +48,20 @@ export default function AccountItemOptionsMenu({
     dispatch(setSnackbarMessage("Address copied to clipboard"))
   }, [address, dispatch])
 
-  const copyKey = useCallback(() => {
-    navigator.clipboard.writeText(key)
+  const copyPrivateKey = async () => {
+    await navigator.clipboard.writeText(key)
     dispatch(setSnackbarMessage("Key copied to clipboard"))
```

Comment:
for mnemonic u wrote:
`dispatch(setSnackbarMessage(sharedT("copyTextSnackbar")))`

maybe we need to do the same for private key?